src/ape/api/networks.py

```diff
@@ -33,7 +33,7 @@ class EcosystemAPI:
     data_folder: Path
     request_header: str
 
-    transaction_class: Type["TransactionAPI"]
+    transaction_types: Mapping["TransactionType", Type["TransactionAPI"]]
```

Comment:
Should this just be a `Dict` type? To be more explicit

Reply:
`mypy` was getting made about something I think, but let me try it again

Reply:
Done!